app/jsx/components/AuthorListComp.jsx

```diff
@@ -1,46 +1,33 @@
 // ##### Author List Component ##### //
 
 import React from 'react'
+import $ from 'jquery'
+import dotdotdot from 'jquery.dotdotdot'
 import { Link } from 'react-router'
 
 class AuthorListComp extends React.Component {
+  componentDidMount() {
+    $('.c-authorlist__list').dotdotdot({watch:'window', after:'.c-authorlist__list-more-link', ellipsis:' ', wrap:'children'});
+  }
+
   handleClick = (e,tab_id) => {
     e.preventDefault()
     this.props.changeTab(tab_id)
   }
 
-  // Add "et al" Link to Author tab if number of authors exceeds 17 
-  // ToDo: This should truncate after certain number of lines, not authors
-  renderExpand = expand => { return(
-    <li>
-    {expand && 
-      <Link href="#" onClick={(e)=>this.handleClick(e, 4)}>et al.</Link>
-    }
-    </li>
-  )}
-
   render() {
     let p = this.props,
-        fulldate = p.pubdate,
-        year = fulldate.match(/\d{4}/),
-        a = p.authors,
-        expand = false
-    if (p.authors && p.authors.length > 17) {
-      a = a.slice(0, 16)
-      expand = true
-    }
-    let authorlist = a.map((author, i, arr) => {
-      let divider = (i < arr.length-1) && <span>;</span>
-      return (
-        <li key={i}><span>{author.name}</span>{divider}</li>
-      )
-    })
+        year = p.pubdate.match(/\d{4}/),
+        authorlist = p.authors.map((author, i) => { return (<li key={i}>{author.name}</li>) })
     return (
       <div className="c-authorlist">
-        <time className="c-authorlist__year">{year && year[0]}</time>
+        {year && <time className="c-authorlist__year">{year[0]}</time> }
         { p.authors && 
             <ul className="c-authorlist__list">      
-              {authorlist} {this.renderExpand(expand)}
+              {authorlist}
+              <li>
+                <Link to="#" className="c-authorlist__list-more-link" onClick={(e)=>this.handleClick(e, 4)}>et al.</Link>
```

Comment:
This et al link is not working for me for some reason. But I'm going to wait to fix until Joel resolves the other bug going on with the et al link showing up no matter how many lines of the authorlist display.